mongodb/connector/connection/connection.go

```diff
@@ -0,0 +1,250 @@
+package mongodbconnection
+
```

Comment:
Also, can this be under "mongodb/connection" instead of "mongodb/connector/connection"

Reply:
I will do this change and try to create our proprietary FE build to check if building the app and running it can work without issues. If not,  I will need to check with Vijay to see if any changes are needed to fe-cli to make this structure work.

Reply:
I have made this change..